performance_tests/Inactive/60Zone1PVAV.idf

```diff
@@ -29,8 +29,10 @@
   HeatBalanceAlgorithm,ConductionTransferFunction;
 
   ShadowCalculation,
-    AverageOverDaysInFrequency,  !- Calculation Method
-    20;                      !- Calculation Frequency
+    PolygonClipping,         !- Shading Calculation Method
```

Comment:
Also - after release - we should revert these (or maybe even remove them altogether @Myoldmopar ?) They aren't current and these changes will fully break them. I guess we try to figure out when they were last working correclyt and re-transition them, but that's low priority.

Reply:
Note: This comment applies only to the files in performance_tests/Inactive.  The active idfs are all good.